src/lib.rs

```diff
@@ -0,0 +1,22 @@
+
+mod args;
+mod audio;
+mod commands;
+mod config;
+mod panic;
+mod render;
+mod scope;
+mod state;
+mod ui;
+
+pub use {
+    args::*,
+    audio::*,
+    commands::*,
+    config::*,
+    panic::*,
+    render::*,
+    scope::*,
+    state::*,
+    ui::*,
+};
```

Comment:
Would prefer just making the modules public instead of reexporting their contents. Will require adding an extra level to all of the function calls in main.rs